lib/reporters/base.js

```diff
@@ -301,11 +302,37 @@ exports.list = function (failures) {
       }
       testTitle += str;
     });
-
     Base.consoleLog(fmt, i + 1, testTitle, msg, stack);
+
+    // Handles formatting and printing Aggregate errors
+    if (test.err.errors) {
+      for (const error of test.err.errors) {
+        const testTitle = addIndent(`Error ${error.message}`, 0);
+        const stack = addIndent(error.stack, 3);
+
+        Base.consoleLog(fmt, error.message, testTitle, '', stack);
```

Comment:
I think the formatting of aggregated errors should use the same formatting as non-aggregated errors, so rather than formatting here locally, send `test.err.errors` to `Base.list` or such?

Helps with:

- Aggregated errors that are nested in aggregated errors
- Aggregated errors with causes in them, pending: https://github.com/mochajs/mocha/pull/4829

Reply:
Good idea! If a test contains an aggregate error it now recursively calls Base.list with those errors.